proposals/0001-simd-process.md

```diff
@@ -158,6 +160,11 @@ far the most effective way to see a proposal through to completion: authors
 should not expect that other project developers will take on responsibility for
 implementing their accepted feature.
 
+### Living
```

Comment:
maybe we should call this "Meta" so folks don't get the wrong impression that partial efforts are acceptable?

Reply:
I was following the standard laid out by [EIP-1](https://eips.ethereum.org/EIPS/eip-1). Given that these would need a separate PR to update their status, we'd be discussing there.

Also I think in general the maintainers(me) need to be reviewing these better to avoid any potential issues of partial efforts being acceptable.

Reply:
got it. yeah that can work. maybe add a blurb about extra review scrutiny or conditions under which this is the right choice

Reply:
Added a blurb about the extra review. Conditions should be covered under `designed to be continually updated and not reach a state of finality`. Tbh I don't expect any SIMD outside of a few select meta SIMDs to ever get here.